src/modules/drive/AddMenu/AddMenu.jsx

```diff
@@ -23,25 +25,27 @@ export const ActionMenuContent = ({
   refreshFolderContent,
   isPublic,
   isEncryptedFolder,
-  displayedFolder
+  displayedFolder,
+  onClick
 }) => {
   const { t } = useI18n()
-  const { isMobile, isDesktop } = useBreakpoints()
+  const { isDesktop } = useBreakpoints()
   const { hasScanner } = useScannerContext()
 
   return (
     <>
-      {isMobile && (
-        <>
-          <Typography variant="h6" className="u-p-1">
-            {t('toolbar.menu_add')}
-          </Typography>
-          <hr />
-        </>
+      <ActionsMenuMobileHeader>
+        <ListItemText
+          primary={t('toolbar.menu_add')}
+          primaryTypographyProps={{ variant: 'h6' }}
+        />
+      </ActionsMenuMobileHeader>
+
+      {canCreateFolder && !isEncryptedFolder && (
+        <AddFolderItem onClick={onClick} />
```

Comment:
I would rather prefer something more explicit than a `onClick` because its always the same action like `onClose` or `onDismiss`

Reply:
Seen together, this is indeed generic, but necessary for the proper functioning of the overload below in the `PublicToolbarMoreMenu.jsx` file.